species/avali.species

```diff
@@ -152,7 +152,34 @@
       { "item" : "avalibench" },
       { "item" : "avalilathe" },
       { "item" : "avalistoragelocker" },
-      { "item" : "mediummetal" }
+      { "item" : "mediummetal" },
+	  { "item" : "avaliavianadventurerpants" },
```

Comment:
Whoops! Such skill (I was too excited and forgot to check, I can go fix that now?).